### ISSUE TITLE
Migrate to vite from create-react-app

### ISSUE THREAD

**Issue by dimaMachina** [codegouvfr/sill#22]
I saw that you use create-react-app, which is outdated for SPA, and I suggest migrating to vite that more flexible and more fastest

**Comment by garronej** [comment]
Agreed,  
The reason we haven't migrated yet is because we use [Keycloakify](https://www.keycloakify.dev/) to integrate the keycloak login pages.  
Currently Keycloakify can only work with Webpack.  
We plan to support vite soon though: https://github.com/keycloakify/keycloakify/issues/271

**Comment by bzg** [comment]
Is this issue still relevant?

**Comment by garronej** [comment]
I mean yes, in the long run it would be better to migrate to Vite since CRA is deprecated but there's really no rush.  
CRA is just the Bunlder, we're still using the latest version of everything otherwise.  

We'll migrate to Vite with Onyxia eventually, I could do the migration for the SILL as well in the meantime.

**Comment by bzg** [comment]
Okay, assigning this issue to the 1.6 milestone.

**Comment by bzg** [comment]
Is it still relevant?

**Comment by garronej** [comment]
Yes still same status but can be closed if we decide we don't care

**Comment by bzg** [comment]
@JeromeBu I'll let you decide if we need to keep this open.

I'm moving this issue outside of the scope of future release for now.

**Comment by garronej** [comment]
At this point there should be no remaining blocker for migrating to vite.  

cra-envs can be replaced by `https://github.com/garronej/vite-envs` and Keycloakify can be removed from the project.
